protocol/tags.go

```diff
@@ -73,7 +73,7 @@ const PingReplyTagMaxSize = 8
 
 // ProposalPayloadTagMaxSize is the maximum size of a ProposalPayloadTag message
 // This value is dominated by the MaxTxnBytesPerBlock
-const ProposalPayloadTagMaxSize = 5247980
+const ProposalPayloadTagMaxSize = 5250301
```

Comment:
There's a discussion elsewhere in the PR where I explain why that can't happen.

In fact, I think this is a bad test because it can never happen.  I'm willing to listen to counterarguments.

Reply:
which other limits would have to be exceeded?